docs/MODULES.md

```diff
@@ -617,6 +618,19 @@ sequencer.loadImage('PATH')
            .addSteps('minify-image')
            .run()
 ```
+# Noise-Reduction
+
+Noise in an image are atypical pixels that are not representing the color or the exposure of the scene correctly. This Noise Reduction module reduces the noise in the image by using different techniques to change the RGB value of the pixels to create a smoother and fuller image.
```

Comment:
Though it is very minor, but please add a space between atypical, (word number 6 on line 623).
Can you somehow reference the different techniques you mentioned above?

Thanks a lot! This is some great work.

Reply:
Hi,
Thank you for your suggestions. I've added references to the techniques onto the module description, just like you asked. However, I don't think there is a need to have a space between atypical, because 'atypical' is a word by itself.
Anyways, I'm glad that I was able to contribute. Please tell me if there are any more changes I need to make. Thanks